### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by gandradep**
This section of the project we added owners and species table. We used the following queries to relate the animals table we already had to the new ones:
- What animals belong to Melody Pond?
MelodyAnimals

-  List of all animals that are Pokemon
pokemons

- List all owners and their animals, remember to include those that don't own any animal
ownersAndAnimas

- How many animals are there per species?
animalPerSpecies

- List all Digimon owned by Jennifer Orwell
digimonJennifer

- List all animals owned by Dean Winchester that haven't tried to escape.
DeanNoEscape

- Who owns the most animals?
moreAnimals

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,4 +11,26 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', 7, TRUE, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '1998-10-13', 3, TRUE, 17);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+/* Data for the owners table */
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
```
Comment (danmainah) [inline data.sql:19]: - [x] @gandradep great work on your entries however you forgot to add data for `Melody Pond`, kindly add it
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @gandradep , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (gandradep) [reply data.sql:19]: @danmainah thank you for the review. I now see I forgot to add that line in the vs code file. I do have it on my database.

Comment (rloterh) [reply data.sql:19]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Gabriel,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉💯🌟 

## Highlights 👍
- Queries properly implemented

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gandradep** [comment]
@rloterh thank you for the review.

**Pull request merged**